internal/artifacts/mtad_test.go

```diff
@@ -25,39 +25,23 @@ var _ = Describe("Mtad", func() {
 		Ω(os.RemoveAll(getTestPath("result"))).Should(Succeed())
 	})
 
-	var _ = Describe("ExecuteGenMtad", func() {
+	var _ = Describe("ExecuteMtadGen", func() {
 		It("Sanity", func() {
-			createDirInTmpFolder("mta", "node-js")
-			Ω(ExecuteGenMtad(getTestPath("mta"), getTestPath("result"), nil, "cf", os.Getwd)).Should(Succeed())
+			Ω(ExecuteMtadGen(getTestPath("mta"), getTestPath("result"), nil, "cf", os.Getwd)).Should(Succeed())
 			Ω(getTestPath("result", "mtad.yaml")).Should(BeAnExistingFile())
 		})
-		It("Fails on creating META-INF folder", func() {
```

Comment:
Please return this test - it checks that if there is a file with the name `META-INF` the generation of `mtad.yaml` fails (because it isn't a folder)